packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -494,7 +494,8 @@ export default function (Commands, Cypress, cy, state, config) {
     })
   }
 
-  function clear (subject, options = {}) {
+  // TODO: change the type of `any` to `Partial<ClearOptions>`
```

Comment:
What is preventing you from making this change now?

Reply:
Check the comment above.